documentation/1.1/content/navigate/image.md

```diff
@@ -24,20 +24,26 @@ in a Kubernetes environment using the [WebLogic Image Tool](https://oracle.githu
 
 ### Design View
 The `Design View` helps you specify the necessary data needed to run the WebLogic Image Tool to build an image for
-running the WebLogic domain with all resources configured and applications deployed. The `Design View` page is divided into `Primary Image` and `Auxiliary Image` pages.
-Auxiliary images are available for the "Model in Image" [domain location]({{< relref "/navigate/project-settings#choosing-a-domain-location" >}}) _only_.
+running the WebLogic domain with all resources configured and applications deployed.
```

Comment:
Unfortunately, this is not strictly accurate in all cases.  When using Domain in PV, the image will never have "all  resources configured and applications deployed" because that data is stored in the Persistent Volume and the Image only has the Java/WebLogic software installed (not unlike the Primary image in the "MII with auxiliary image" use case).   Maybe we should just remove the "with all resources configured and applications deployed" to make it fit all use cases?

Reply:
ok, done.